src/main/java/lotto/domain/Ranking.java

```diff
@@ -0,0 +1,62 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Ranking {
+    FIRST(6, 0, 2000000000),
+    SECOND(5, 1, 30000000),
+    THIRD(5, 0, 1500000),
+    FOURTH(4, 0, 50000),
+    FIFTH(3, 0, 5000),
+    OTHERS(0, 0, 0);
+
+    private final int numberOfMatches;
+    private final int prizeMoney;
+    private final int bonus;
+    private int count = 0;
+
+    Ranking(final int numberOfMatches, int bonus, final int prizeMoney) {
+        this.numberOfMatches = numberOfMatches;
+        if (numberOfMatches == 5) {
+            this.bonus = bonus;
+        } else {
+            this.bonus = 0;
+        }
+        this.prizeMoney = prizeMoney;
+    }
+
+    public static Ranking of(final int numberOfMatches, final int bonus) {
+        return Arrays.stream(values())
```

Comment:
`of` 메서드를 받을 때도 어짜피 `bonus` 일치 여부를 입력 받아야는데  Ranking의 상태 값으로 `bonus`를 둘 것이 아니라, 아래와 같이 변경해볼 수 있지 않을까요? 😃 
```suggestion
    public static Ranking of(final int numberOfMatches, final boolean matchBonusNumber) {
        if (numberOfMatches == SECOND.numberOfMatches && !matchBonusNumber) {
            return THIRD;
        }
        
        return Arrays.stream(values())
```